### DISCUSSION TITLE
Have "humanhusk"(s) spawn instead of "husk"(s).

### DISCUSSION THREAD

**Discussion by MarkCringan** [FakeFishGames/Barotrauma#13475] [Ideas and suggestions]
[IGNORE THIS PARAGRAPH] (See comment by AzureNocte) With human-infected husks being in the game plus the addition of the new broken diving suit, there does not really seem to be a point in the standard husk enemy anymore. 

There are some instances where spawning a standard husk instead of a husk human does not make any sense, such as the abandoned outpost mission that requires your crew to clear out cultists and husks. In the context of the mission, why are there so many +[old and decrepid] -[people wearing old] diving suits?

Additionally, husk-humans have the ability to, for lack of a better term, """resurrect""" themselves which would make them more "entertaining" to play against. 

I would love to hear any feedback that you all have on this idea. Thank you!

I did not see this idea proposed anywhere else, but if it already had been then sorry lol.

**Comment by AzureNocte** [comment]
Husks are the final form of husk infection, lore-wise they are just diving suit controlled by husk parasite, there are little to none human parts left inside, so they are different to human husks, which just transformed.

**Reply by MarkCringan** [reply]
Huh, I did not know that. That is fascinating. If the reanimated diving suit is considered to be the last stage, don't you think it would make more sense for some instances, like clearing out the cultists extremists, to use human-husks rather than "final form" husks?

**Comment by bookslayer10** [comment]
What happens when a human is husked without a diving suit on and is left alone for years? They can't turn into a husk because they can't get a diving suit  on, so what happens to them? Husked crawlers don't need a suit and appear to have most of their muscle mass intact. 

There should be husked humans without dive suits naturally spawning for this reason, just for some more variety.
